app/Http/Controllers/WikiController.php

```diff
@@ -108,6 +116,8 @@ public function create(Request $request): \Illuminate\Http\Response
         $this->dispatch(new ProvisionWikiDbJob(null, null, 10));
         $this->dispatch(new ProvisionQueryserviceNamespaceJob(null, 10));
 
+        $this->dispatch(new ElasticSearchIndexInit($wiki->domain));
```

Comment:
@toban 
Do we need to create this class somewhere ?

Reply:
i guess yes. since psalm is failing the lint test

Reply:
Looks like we might just need to import it!